lib/config.ts

```diff
@@ -80,7 +80,7 @@ class Config {
     this.enableErrLogging = !!process.env.ENABLE_ERR_LOGGING;
 
     // New codebase
-    this.newUIcodebase = process.env.ENV_TYPE == 'stage-new-ui';
+    this.newUIcodebase = process.env.ENV_TYPE != 'production';
```

Comment:
I gues it would be driven by URL and not env type for now. if url is qaproauth then inew ui should be used. cc @Frawless

Reply:
or leave it as is, seems prod will be switched to new-ui soon

Reply:
I don't have strong opinion as we will remove it soon after prod will be migrated

Reply:
I would let it as it is and remove whole switching part after pushing new ui to production